### ISSUE TITLE
Use XAML 2009 as the basis for XAML Standard

### ISSUE THREAD

**Issue by bondsbw** [Microsoft/xaml-standard#45]
Please use XAML 2009 as the basis of XAML Standard, instead of XAML 2006.

**Comment by Mike-EEE** [comment]
Technically Xamarin.Forms Xaml model already does this. :)

**Comment by bondsbw** [comment]
WPF XAML does not.  (#20)

**Comment by PedroLamas** [comment]
Generics in XAML? Yes, please!!!! 😁

**Comment by harinikmsft** [comment]
What are the most sorely missed features from xaml2009 for folks? We are looking into adding more XAML language features to UWP (and XAML Standard). 

Some like x:TypeArguments, x:Array, x:Type are really good features with no real alternatives today. Some others like x:Static, x:Reference - you can achieve most scenarios through x:Bind (if x:Bind makes it to XAML Standard). 

Tell us more about what you miss (especially if you like that Xamarin.Forms xaml supports it).

**Comment by PedroLamas** [comment]
I can live without the rest, but I have in the past felt the need for `x:TypeArguments` for generics in XAML.

Obviously, even for that one, there are workarounds, but that goes against the point of being able to use it directly!

**Comment by james1301** [comment]
Yes x:TypeArguments is a big must. WPF and UWP are miles behind without it, compared to Xamarin Forms.
